File: app/src/main/java/com/cyberIyke/allvideodowloader/interfaces/ShortcutListner.java
package com.cyberIyke.allvideodowloader.interfaces;

import com.cyberIyke.allvideodowloader.database.ShortcutTable;

public interface ShortcutListner {
    void shortcutClick(ShortcutTable shortcutTable);
    void shortcutRemoveClick(ShortcutTable shortcutTable);
}

File: app/src/main/java/com/cyberIyke/allvideodowloader/interfaces/ShortcutListner.kt
package com.cyberIyke.allvideodowloader.interfaces

import com.cyberIyke.allvideodowloader.database.ShortcutTable

open interface ShortcutListner {
    fun shortcutClick(shortcutTable: ShortcutTable?)
    fun shortcutRemoveClick(shortcutTable: ShortcutTable?)
}